__tests__/page.test.tsx

```diff
@@ -1,9 +1,31 @@
-import { Header } from '../src/components/Header/Header';
 import { render, screen } from '@testing-library/react';
+import { Header } from '../src/components/Header/Header';
+import { PromoBanner } from '../src/components/promo-banner/promo-banner';
+import React, { ClassAttributes, ImgHTMLAttributes } from 'react';
+
+jest.mock('next/image', () => ({
+  __esModule: true,
+  default: (
+    props: JSX.IntrinsicAttributes &
+      ClassAttributes<HTMLImageElement> &
+      ImgHTMLAttributes<HTMLImageElement>,
+  ) => <div />,
+}));
 
 describe('Header', () => {
   it('should pass all the tests', () => {
     render(<Header />);
     expect(screen.getByText('Главная')).toBeInTheDocument();
   });
 });
+
+describe('Promo Banner', () => {
```

Comment:
Better to name `describe` blocks with component names. In this case - `PromoBanner`. This wat it will be easier to copy this name and find it in the code